rschuma2/OGS_RollBall
# NB Update 1: Added new 4x new pickup items (coin, jewels) along with materials

**To Test:**
Navigate to _**Content\Meshes\PickupMeshes**_ and ensure 4x new meshes are present
Navigate to **_Content\Materials\PickupMaterials_** and ensure 5x materials are present

**Pickups without Materials**
![Update 1 - Pickup Mesh](https://user-images.githubusercontent.com/29407608/80261617-8bd94880-8650-11ea-80f8-9fd840527fd5.JPG)

**Pickups with Materials**
![Update 1 - Pickup Mesh with Color](https://user-images.githubusercontent.com/29407608/80261625-90056600-8650-11ea-820f-065189b20c8b.JPG)

Path to Meshes: **_Content\Meshes\PickupMeshes_**
![Update 1 - Pickup Mesh File Path](https://user-images.githubusercontent.com/29407608/80261635-972c7400-8650-11ea-89f4-a1532f181783.JPG)


Path to Materials: **_Content\Materials\PickupMaterials_**
![Update 1 - Pickup Material File Path](https://user-images.githubusercontent.com/29407608/80261638-98f63780-8650-11ea-849d-0176e03688f3.JPG)
